universe-base Docker image fails to build

`sudo make base` fails with the following error:

```bash
rm -rf certs &&							\
rm -f local-universe.tar.gz || 0
mkdir certs && openssl req					\
	-newkey rsa:4096 -nodes -sha256 -keyout certs/domain.key	\
	-x509 -days 365 -out certs/domain.crt			\
	-subj "/CN=master.mesos"
Generating a 4096 bit RSA private key
.............................................................................................................................................++
....................................++
writing new private key to 'certs/domain.key'
-----
docker build -t universe-base -f Dockerfile.base .
Sending build context to Docker daemon 23.04 kB
Step 1 : FROM registry:2.4.1
 ---> 8ff6a4aae657
Step 2 : ENV NGINX_VERSION 1.10.1-1~jessie
 ---> Using cache
 ---> d39783daa740
Step 3 : RUN apt-key adv --keyserver hkp://pgp.mit.edu:80 --recv-keys 573BFD6B3D8FBC641079A6ABABF5BD827BD9BF62   && echo "deb http://nginx.org/packages/debian/ jessie nginx" >> /etc/apt/sources.list   && apt-get update   && apt-get install --no-install-recommends --no-install-suggests -y             ca-certificates             nginx=${NGINX_VERSION}             nginx-module-xslt             nginx-module-geoip             nginx-module-image-filter             nginx-module-perl             nginx-module-njs             gettext-base   && rm -rf /var/lib/apt/lists/*
 ---> Running in 5b0df9d47a86
Executing: gpg --ignore-time-conflict --no-options --no-default-keyring --homedir /tmp/tmp.nRdI9c7QRJ --no-auto-check-trustdb --trust-model always --primary-keyring /etc/apt/trusted.gpg --keyring /etc/apt/trusted.gpg.d/debian-archive-jessie-automatic.gpg --keyring /etc/apt/trusted.gpg.d/debian-archive-jessie-security-automatic.gpg --keyring /etc/apt/trusted.gpg.d/debian-archive-jessie-stable.gpg --keyring /etc/apt/trusted.gpg.d/debian-archive-squeeze-automatic.gpg --keyring /etc/apt/trusted.gpg.d/debian-archive-squeeze-stable.gpg --keyring /etc/apt/trusted.gpg.d/debian-archive-wheezy-automatic.gpg --keyring /etc/apt/trusted.gpg.d/debian-archive-wheezy-stable.gpg --keyserver hkp://pgp.mit.edu:80 --recv-keys 573BFD6B3D8FBC641079A6ABABF5BD827BD9BF62
gpg: requesting key 7BD9BF62 from hkp server pgp.mit.edu
gpg: key 7BD9BF62: public key "nginx signing key <signing-key@nginx.com>" imported
gpg: Total number processed: 1
gpg:               imported: 1  (RSA: 1)
Get:1 http://security.debian.org jessie/updates InRelease [63.1 kB]
Ign http://nginx.org jessie InRelease
Get:2 http://security.debian.org jessie/updates/main amd64 Packages [398 kB]
Get:3 http://nginx.org jessie Release.gpg [287 B]
Get:4 http://nginx.org jessie Release [2325 B]
Get:5 http://httpredir.debian.org jessie-updates InRelease [142 kB]
Get:6 http://nginx.org jessie/nginx amd64 Packages [5458 B]
Ign http://httpredir.debian.org jessie InRelease
Get:7 http://httpredir.debian.org jessie-updates/main amd64 Packages [17.6 kB]
Get:8 http://httpredir.debian.org jessie Release.gpg [2373 B]
Get:9 http://httpredir.debian.org jessie Release [148 kB]
Get:10 http://httpredir.debian.org jessie/main amd64 Packages [9064 kB]
Fetched 9844 kB in 27s (356 kB/s)
Reading package lists...
Reading package lists...
Building dependency tree...
Reading state information...
ca-certificates is already the newest version.
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 nginx-module-geoip : Depends: nginx (= 1.10.2-1~jessie)
 nginx-module-image-filter : Depends: nginx (= 1.10.2-1~jessie)
 nginx-module-njs : Depends: nginx (= 1.10.2-1~jessie)
 nginx-module-perl : Depends: nginx (= 1.10.2-1~jessie)
 nginx-module-xslt : Depends: nginx (= 1.10.2-1~jessie)
E: Unable to correct problems, you have held broken packages.
The command '/bin/sh -c apt-key adv --keyserver hkp://pgp.mit.edu:80 --recv-keys 573BFD6B3D8FBC641079A6ABABF5BD827BD9BF62   && echo "deb http://nginx.org/packages/debian/ jessie nginx" >> /etc/apt/sources.list   && apt-get update   && apt-get install --no-install-recommends --no-install-suggests -y             ca-certificates             nginx=${NGINX_VERSION}             nginx-module-xslt             nginx-module-geoip             nginx-module-image-filter             nginx-module-perl             nginx-module-njs             gettext-base   && rm -rf /var/lib/apt/lists/*' returned a non-zero code: 100
Makefile:11: recipe for target 'base' failed
make: *** [base] Error 100
```

This is probably due to a newer version of nginx than 1.10.1-1~jessie. Is there a particular reason to require this version?

--- a/docker/local-universe/Dockerfile.base
+++ b/docker/local-universe/Dockerfile.base
@@ -1,6 +1,6 @@
 FROM registry:2.4.1
 
-ENV NGINX_VERSION 1.10.1-1~jessie
+ENV NGINX_VERSION 1.10.2-1~jessie
 
 RUN apt-key adv --keyserver hkp://pgp.mit.edu:80 --recv-keys 573BFD6B3D8FBC641079A6ABABF5BD827BD9BF62 \
   && echo "deb http://nginx.org/packages/debian/ jessie nginx" >> /etc/apt/sources.list \

